pkg/model/initializers.go

```diff
@@ -304,18 +304,19 @@ func (ml *ModelLoader) grpcModel(backend string, o *Options) func(string, string
 					return nil, fmt.Errorf("failed allocating free ports: %s", err.Error())
 				}
 				// Make sure the process is executable
```

Comment:
This comment is probably duplicated from new line 60 of process.go - not worth restarting the build process over, but I'll try to remember to clean it up next time :)